modules/appeals_api/app/swagger/higher_level_reviews/v0/swagger_dev.json

```diff
@@ -704,17 +704,17 @@
               }
             }
           },
```

Comment:
These changes to the HLR docs are here because I reordered the 404 and 403 examples so that they'd be in increasing number order